api/python/ai/chronon/repo/gitpython_utils.py

```
@@ -0,0 +1,14 @@
+from typing import Optional
+
+from git import Repo
+
+
+def get_default_origin_branch(path, repo: Optional[Repo] = None):
+    if not repo:
+        repo = Repo(path, search_parent_directories=True)
+    return repo.remotes.origin.refs.HEAD.reference.name.split('/')[-1]
+
+def get_current_branch(path, repo: Optional[Repo] = None):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)